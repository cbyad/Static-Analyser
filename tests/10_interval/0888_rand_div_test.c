{
  int x;
  x = rand(5,10) / rand(-1,1);
  print(x);
}