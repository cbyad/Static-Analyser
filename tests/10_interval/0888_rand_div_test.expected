tests/10_interval/0888_rand_div_test.c:4.2-11: [ x in [-10;10] ]
analysis ended